docker-compose.yaml

```diff
@@ -15,5 +15,6 @@ services:
       - /var/run/docker.sock:/tmp/docker.sock:ro
       - $HOME/.autok3s/:$HOME/.autok3s/
     environment:
-    - AUTOK3S_CONFIG=$HOME/.autok3s/
-    - VIRTUAL_HOST=autok3s.vcap.me
+      - AUTOK3S_CONFIG=$HOME/.autok3s/
+      - VIRTUAL_HOST=autok3s.vcap.me
+      - DOCKER_HOST=unix:///tmp/docker.sock
```

Comment:
Instead of Using environment variable to specify the `DOCKER_HOST`, changing the docker.sock mount point in container would be better.

Reply:
Done